crawler/plugins/applications/util.py

```diff
@@ -0,0 +1,63 @@
+import dockerutils
+
+
+class Inspector(object):
+    '''
+    Interface class.
+    '''
+
+    def __init__(self, container_id):
+        self.inspect = dockerutils.exec_dockerinspect(container_id)
+        # self.cotainers = dockerutils.exec_dockerlistcontainers()
```

Comment:
You might have left this comment by mistake.

Would you prefer to add these methods: `is_app_container`, `get_ip`, `get_ports` to the DockerContainer class (dockercontainer.py)?